scripts/glusterfs_vol.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+HOSTLIST=$1
+STRIPE=$2
+REPLICA=$3
+
+GLUSTERFS_VOL_NAME=gv0
+
+mkdir -p /mnt/brick1/$GLUSTERFS_VOL_NAME
```

Comment:
/mnt/brick1 is hardcoded, can you add it as a param and default to that value ?

Reply:
Added BRICK_MOUNT_PT variable with default /mnt/brick1